### PR TITLE
Corrected information about banner location

### PR DESCRIPTION
**Pull request by tknippenberg**

### CODE REVIEW

File: content/en/docs/developerportal/general/team-server/migrate-to-git.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ We are working on solutions to both notify you when this issue applies to you an
 
 ## 3 How Migration Works
 
-If your app is eligible for migration and your team role is Scrum Master, you see a notification on possible migration at the top of all pages within the **Collaborate** section. Click **Migrate to Git** to start migration:
+If your main line ('main') branch is Mendix version 9.24.11 or above, and your team role is Scrum Master, you see a notification on possible migration at the top of the **Team Server** page for your app. Click **Migrate to Git** to check for migration eligibility and to start the migration:
 
 {{< figure src="/attachments/developerportal/general/team-server/migrate-to-git/migrate-to-git.jpg" >}}
```
Comment (MariaShaposhnikova) [inline content/en/docs/developerportal/general/team-server/migrate-to-git.md]: @tknippenberg , does the screenshot need an update too?

Comment (MariaShaposhnikova) [reply content/en/docs/developerportal/general/team-server/migrate-to-git.md]: I tried reproducing it myself, but could not.

Comment (tknippenberg) [reply content/en/docs/developerportal/general/team-server/migrate-to-git.md]: The screenshot is technically outdated, but it it's so different that it'll confuse the customer. If you are on the Team Server page, it is very clear when you have the banner (or not).

To reproduce:
- Create an SVN app on Team Server
- Ensure the SVN app is 9.24.11
- Go to Team Server page in the Developer Portal for your app

Comment (MariaShaposhnikova) [reply content/en/docs/developerportal/general/team-server/migrate-to-git.md]: @tknippenberg , not to confuse cumstomers, I've deleted the image.

**Pull request merged**